src/datasources/balances-api/zerion-balances-api.service.ts

```diff
@@ -98,7 +98,8 @@ export class ZerionBalancesApi implements IBalancesApi {
     if (cached != null) {
       const { key, field } = cacheDir;
       this.loggingService.debug({ type: 'cache_hit', key, field });
-      return this._mapBalances(chainName, JSON.parse(cached));
+      const zerionBalances: ZerionBalance[] = JSON.parse(cached);
```

Comment:
Looking at this in the diffs (below https://github.com/safe-global/safe-client-gateway/pull/1533/files#r1624713015), I think we should also add a comment here regarding a guard.

Reply:
Done in [f822c56](https://github.com/safe-global/safe-client-gateway/pull/1533/commits/f822c56da6fa82c7cf215ace689edd546dd5e108)